examples/example_plugin/example_plugin/jobs.py

```diff
@@ -130,3 +130,4 @@ def receive_job_button(self, obj):
     ExampleSimpleJobButtonReceiver,
     ExampleComplexJobButtonReceiver,
 )
+register_jobs(*jobs)
```

Comment:
I think it would good to align the patterns here and in files dropped into `JOBS_ROOT`.

For example if this `jobs` attribute were to just call `register_jobs()` on them internally, then the same pattern used in an App's `jobs.py` or a Git repo, would now be asserted in file-based jobs as well.

The only other consideration would be to obviate this entirely and use a meta class on `BaseJob` to automatically register any subclass into the Celery task registry.

Reply:
I think we should align on `register_jobs()` and use the celery task registry instead of the `jobs` variable to register app jobs in the database. This tracks with our earlier discussion on standardizing the natural keys for the job database model and celery task registry.